docs/usage-with-routing.stories.mdx

```diff
@@ -0,0 +1,131 @@
+<Meta title="Documentation/Usage with routing" />
```

Comment:
We can ignore this story from chromatic.

Reply:
I don't think these docs ones get added to chromatic, at least I didn't see one getting added in the UI review check, and none of the others have ignore on either

Reply:
Ah yeah, because it doesn't have `<Story>`. Sorry about the confusion.